exe/wallet/Main.hs

```diff
@@ -130,6 +143,11 @@ Options:
   --bridge-port <INT>         port used for communicating with the http-bridge [default: 8080]
   --address-pool-gap <INT>    number of unused consecutive addresses to keep track of [default: 20]
   --size <INT>                number of mnemonic words to generate [default: 15]
+  --payment <PAYMENT>         address to send to and amount to send separated by @: '<amount>@<address>'
+
+Examples:
+  cardano-wallet transaction create --wallet-id 2512a00e9653fe49a44a5886202e24d77eeb998f --payment 22@2cWKMJemoBam7gg1y5K2aFDhAm5L8fVc96NfxgcGhdLMFTsToNAU9t5HVdBBQKy4iDswL # Create a transaction and send 22 lovelace from wallet-id to specified addres
```

Comment:
What about: 

- Putting the `--payment` on a new line with a `\` 
- Using either a Yoroi address, or an arbitrary shorter string in base58, just to convey the idea, it doesn't have to be a real address
- Make sure the last digit of the amount isn't the same digit as the address :sweat_smile: ... for enhanced readability

Reply:
> Make sure the last digit of the amount isn't the same digit as the address sweat_smile ... for enhanced readability

makes sense - didn't notice

> Putting the --payment on a new line with a \

:+1: 

> Using either a Yoroi address, or an arbitrary shorter string in base58, just to convey the idea, it doesn't have to be a real address

:+1: